simple/cm_data.c

```diff
@@ -191,6 +191,19 @@ static int server_accept(size_t paramlen)
 	}
 
 	fi_shutdown(ep, 0);
+	memset(entry, 0, sizeof(*entry) + paramlen);
+	ret = fi_eq_sread(eq, &event, entry, sizeof(*entry), -1, 0);
```

Comment:
It looks odd seeing memset use sizeof(*entry) + paramlen, but sread only use sizeof(*entry)